### DISCUSSION TITLE
编辑器：换行时，缩进希望保持和上一行一样而不是跑到开头

### DISCUSSION THREAD

**Discussion by QING925** [linuxdeepin/developer-center#2848] [Features Request & Ideas | 特性请求 & 头脑风暴]
希望多一个功能，换行的时候，缩进可以保持和上一行一样，而不是跑到开头。

**Comment by guoshaoyu** [comment]
> 希望多一个功能，换行的时候，缩进可以保持和上一行一样，而不是跑到开头。

deepin-editor是一个文本记录编辑的工具，您的需求更符合作为一个开发者编写代码时的需求，但是软件工具还是需求考虑到普通用户的基本需求，纯粹用文本去编写记录事物的时候，更多的普通用户还是习惯性的觉得换行就是简单的换行直接到下一行的首位；
当然，您的建议我们也是会当前新的需求作为评审评估；
